.github/workflows/reusable-misc-tests.yml

```diff
@@ -165,13 +165,14 @@ jobs:
                 --build-arg DOCKERHUB_MIRROR=registry-1.docker.io.mirror.corp.earthly.dev \
              +test --GLOBAL_WAIT_END=true
         if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
-      - name: Execute +test using wait-block override (Fork Only)
-        uses: nick-fields/retry@v2
-        with:
-          timeout_minutes: 3
-          max_attempts: 3
-          command: ${{inputs.SUDO}} ${{inputs.BUILT_EARTHLY_PATH}} --ci -P --global-wait-end +test --GLOBAL_WAIT_END=true
-        if: github.event_name != 'push' && github.event.pull_request.head.repo.full_name != github.repository
+        # TODO: ensure the target being run can work without docker credentials
+#      - name: Execute +test using wait-block override (Fork Only)
+#        uses: nick-fields/retry@v2
+#        with:
+#          timeout_minutes: 3
+#          max_attempts: 3
+#          command: ${{inputs.SUDO}} ${{inputs.BUILT_EARTHLY_PATH}} --ci -P --global-wait-end +test --GLOBAL_WAIT_END=true
+#        if: github.event_name != 'push' && github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
I would maybe remove this altogether - we have git history if we ever want to resurect this piece of code.

Reply:
done